styles.css

```
@@ -8,24 +10,518 @@
+    /*Lichtgrijs #F3F6F9 (voor de achtergrond van de projecten)*/
+    /*Grijs #7A7A7A (voor de service-tegel tekst);*/
+}
+
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
 
+header {
+    position: relative;
+    color: white;
+    background: var(--gradient);
+    padding: 100px 0;
+}
+
+header img {
+    position: absolute;
+    bottom: 0;
+    width: 50px;
+}
+
+header p {
+    font-weight: 300;
+    margin: 20px 0;
+}
+
+h1 {
+    margin: 0 0;
+    font-size: 35px;
+    font-weight: 500;
+    text-decoration-style: dotted;
+}
+
+header button {
+    width: 200px;
+    height: 60px;
+    margin: 0 0 50px 0;
+    border-radius: 30px 30px;
+    border: none;
+    color: white;
+    text-transform: uppercase;
+    font-size: 13px;
+    font-weight: 500;
+    background-color: #ED4385;
+    box-shadow: 1px 1px 5px black;
+}
+
+
+@media screen and (min-width: 576px) {
+    header button {
+        width: 180px;
+        height: 45px;
+        margin: 0 0 50px 0;
+        border-radius: 25px 25px;
+        border: none;
+        color: white;
+        text-transform: uppercase;
+        font-size: 11px;
+        font-weight: 300;
+        background-color: #ED4385;
+        box-shadow: 1px 1px 5px black;
+    }
+}
+
+.driekhoek-header {
+    width: inherit;
+    height: 0;
+    border-image: var(--gradient);
+    /*border-left: 100px solid green;*/
+    border-right: 100vw solid blue;
+    border-bottom: 150px solid transparent;
+    clip-path: polygon(100% 0%, 0% 0%, 100% 100%);
+
+    /*border-top: 100px solid red;*/
+
+}
+
+nav {
+    background: var(--gradient);
+    padding: 10px 10px 0 0;
+}
+
+nav ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    gap: 35px;
+}
+
+@media screen and (min-width: 576px) {
+    nav ul {
+        list-style-type: none;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        gap: 35px;
+    }
+}
+
+
+ul a {
+    color: white;
+    text-decoration: none;
+    font-weight: 300;
+    font-size: 14px;
+}
+
 /* --------------- 4.2 Introduction ------------------- */
 
-/* ------------------------------- 5. Media Queries -------------------------------- */
\ No newline at end of file
+.section1 {
+    padding: 50px 0 50px 0;
+}
+
+.section1 p {
+    color: #4845E4;
+    padding: 10px 0 10px 0;
+    font-weight: 500;
+    font-size: 23px;
+    font-family: 'Merriweather', serif;
+}
+
+#introduction-question {
+    color: black;
+    padding: 50px 0 0 0;
+    font-size: 14px;
+    font-weight: 300;
+}
+
+#introduction-question a {
+    text-decoration: none;
+}
+
+/* ------------------------------- 4.3 Projects -------------------------------- */
+
+.section-article {
+    background-color: #F3F6F9;
+    padding: 100px 0 100px 0;
```

Je kan dit korter doen
```suggestion
    padding: 100px 0;
```